### PR TITLE
Update AlloyUtils.java

### PR DESCRIPTION
**Pull request by R0ckyZhu**
fix until bug - allow set names containing "_"

### CODE REVIEW

File: _tmp_musical-chairs-traces.als
Comment (nancyday) [inline _tmp_musical-chairs-traces.als:1]: Should this file be part of the commit?

File: farmer-traces.als
Comment (nancyday) [inline farmer-traces.als:1]: Should this file be part of the commit?

**Review by nancyday — CHANGES_REQUESTED** [review]
There seem to  be files here that should not be part of the commit.

File: _tmp_musical-chairs-traces.als
Comment (R0ckyZhu) [reply _tmp_musical-chairs-traces.als:1]: No. I was updating my aldb repo and didn't mean to commit the changes.

**Pull request closed without merging**